h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -1158,80 +1162,56 @@ private Chunk chk_xnew(Chunk[] chks, int c) {
           if (Double.isNaN(a[j])) continue;   // Skip missing observations in row
 
           // Calculate x_i * Y_j where Y_j is sub-matrix corresponding to categorical col j
-          double[] xy = new double[_yt._numLevels[j]];
-          for (int level = 0; level < xy.length; level++) {
+//          for (int level = 0; level < xy.length; level++) {
```

Comment:
Still trying to pass Jenkins test and don't want to confuse myself.  Once my Jenkins are green.  They will be gone.